app/views/two_factor_authentication/otp_verification/show.html.slim

```diff
@@ -9,12 +9,16 @@ p == @presenter.phone_number_message
   = label_tag 'code', \
     t('simple_form.required.html') + t('forms.two_factor.code'), \
     class: 'block bold'
-  .col-12.sm-col-5.mb4.sm-mb0.sm-mr-20p.inline-block
+  .col-12.sm-col-5.mb2.sm-mb0.sm-mr-20p.inline-block
     = text_field_tag(:code, '', value: @presenter.code_value, required: true,
       autofocus: true, pattern: '[0-9]*', class: 'col-12 field monospace mfa',
       'aria-describedby': 'code-instructs', maxlength: Devise.direct_otp_length,
       autocomplete: 'off', type: 'tel')
   = submit_tag t('forms.buttons.submit.default'), class: 'btn btn-primary align-top'
+  - if @presenter.remember_device_available?
+    .border.col-9.rounded-lg.mt2
+      = check_box_tag 'remember_device', true, false, class: 'my2 ml2 mr1'
+      = label_tag 'remember_device', 'Remember this device', class: 'blue'
```

Comment:
Should this say how many days the device will be remembered?

Reply:
What do ya think @micahtaylor?

Reply:
That would be ideal. The mock has it listed as `a month` - https://cm-jira.usa.gov/secure/attachment/55472/03e%20-%202fa%20-%20enter%20code%20%26%20remember%20device.png
Is that accurate?

Reply:
Yep, missed that. What do you think of "Remember this computer for n days"?

Reply:
Is it `30 days`? If so, I'm fine just generalizing it to `Remember this computer for one month`

Reply:
The default is 30 days, but it is configurable, so we can change it without having to make a code change. It would be nice if changing the config to also update the text here. That's a bit easier if we use days.

If it is important, we can probably build a smarter helper that figures out what unit of time to use based on the number.